### PR TITLE
local acf json

### PR DESCRIPTION
**Pull request by zackkrida**
fixes one of the tasks in #3. the build keeps giving me hell. so aggressive and for whatever reason not agreeing with my local testing

Edit: I'm dumb. Sorry for all the commits.

### CODE REVIEW

**Comment by zackkrida** [comment]
:fire: :hankey: i guess this is just never gonna fly. damn

File: sanders.sublime-workspace [hunk 1]
```diff
@@ -0,0 +1,319 @@
+{
```
Comment (atticoos) [inline sanders.sublime-workspace]: :fire:

Comment (atticoos) [inline sanders.sublime-workspace]: This file shouldn't be in here

**Comment by atticoos** [comment]
:+1: thanks!

**Pull request merged**